src/ralph/licences/models.py

```diff
@@ -36,8 +37,11 @@ class LicenceType(NamedMixin, models.Model):
     def create_from_string(cls, string_name, *args, **kwargs):
         return cls(name=string_name)
 
+    class Permissions:
+        blacklist = set()
```

Comment:
do we really need to add this to every class which inherits from PermByFieldMixin?